_data/blogs.yml

```diff
@@ -1,3 +1,8 @@
+- name: Ainesh Sootha
+  blog: //aineshsootha.wordpress.com/
+  blog_img: AineshSoothaBlog.png
+  article: Google Code-in and the Open Tech community.
+  article_link: https://aineshsootha.wordpress.com/2016/12/10/google-code-in-and-the-open-tech-community/
```

Comment:
* Please add your changes at the end of each file not in the beginning.
* Squash all your commits into one and rename it to something meaningful, such as "Adding <-your-name-> in students/mentors section"
* Also provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.